bin/raptor

```diff
@@ -173,4 +173,18 @@ parent
     raptor(options);
   });
 
+parent
+  .command('submit')
+  .help('Submit a Raptor JSON log to an InfluxDB database')
+  .callback(function(options) {
```

Comment:
This is inconsistent with the 'test' command. Shouldn't there just be an option for the log file path, in position 1? So you can do 'raptor submit raptor.log'? Instead of 'raptor submit --log raptor.log'